src/components/attendee/SearchOverlay/SpaceItem/SpaceItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "components/attendee/Button";
+
+import { SPACE_TAXON } from "settings";
+
+import { SpaceWithId } from "types/id";
+
+import { enterSpace } from "utils/url";
+
+import { useWorldById } from "hooks/worlds/useWorldById";
+
+import CN from "../SearchOverlay.module.scss";
+
+export type SpaceItemProps = {
+  space: SpaceWithId;
+  onClick: () => void;
+};
+
+export const SpaceItem: React.FC<SpaceItemProps> = ({ space, onClick }) => {
+  const { world } = useWorldById({ worldId: space.worldId });
+
+  const { push: openUrlUsingRouter } = useHistory();
+
+  const handleEnterSpace = useCallback(() => {
+    enterSpace(world?.slug, space?.slug, {
+      customOpenRelativeUrl: openUrlUsingRouter,
+    });
+    onClick();
+  }, [world?.slug, openUrlUsingRouter, onClick, space?.slug]);
+
+  const { subtitle, description } = space?.config?.landingPageConfig ?? {};
```

Comment:
nope, that would lead to destructuring error 
<img width="800" alt="image" src="https://user-images.githubusercontent.com/56254806/169767388-834c37dc-3a08-45ba-95a0-e9dc532ca52a.png">